packages/headless/esbuild.mjs

```diff
@@ -190,10 +190,6 @@ async function buildBrowserConfig(options, outDir) {
     plugins: [
       alias({
         'coveo.analytics': resolveEsm('coveo.analytics'),
-        '@coveo/please-give-me-fetch': resolve(
-          './ponyfills',
-          'fetch-ponyfill-browser.js'
```

Comment:
In think until we figure out why we needed to polyfill fetch for a browser, we shouldn't be confident to remove it?